### PR TITLE
Changed routing file name

### PR DESCRIPTION
**Pull request by imSmaity**
## Check List (Check all the boxes which are applicable) 

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] All new and existing tests passed.
- [ ] I have updated the documentation accordingly.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by chandankumar1307** [comment]
Create an issue and link your PR with it

File: src/routes/Routing.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+function Redirect({to}){
+  const navigate=useNavigate()
+  useEffect(()=>{
+    navigate(to)
+  },[navigate,to])
+
+  return;
+}
+
+function Routing() {
```
Comment (chandankumar1307) [inline src/routes/Routing.js:17]: Use arrow functions to keep the logic the same throughout the codebase.

**Review by chandankumar1307 — APPROVED** [review]
Do these changes and commit

File: src/routes/Routing.js (hunk 1 above)
Comment (imSmaity) [reply src/routes/Routing.js]: Now I have changed this.

**Review by AayushShuk1a — APPROVED** [review]
LGTM👍

**Pull request merged**